src/components/players/details/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+
+import Placeholder from './card/placeholder.js';
+import Card from '../_shared/card';
+import QuickStats from './quickStats/';
+import Games from './games/index.js';
+import Stats from './stats';
+
+
+import styles from './index.module.scss';
+
+
+const Details = ({ player, teams, games }) => {
+    if (!player) {
+        return (
+            <div className={styles.container}>
+                <Placeholder />
+            </div>
+        );
+    }
+
+    const quickStats = {
+        ppg: player.ppg, 
+        reb: player.totReb, 
+        ast: player.assists, 
+        "fg %": player.fgp, 
+    }
```

Comment:
Though I understand the logic you went for here having the key as the "title" and the value as the "value". It's not the most efficient or pretty.

the better way is too:

```
const quickStats = [
  { title: 'ppg', value: 'player.ppg' },
  { title: 'reb', value: player.totReb },
  ...
];
```

and obviously you'll have to change the way your shared components work. But you get the idea